app/viewmodels/helpers/events/MovementEventHelper.scala

```diff
@@ -459,4 +471,235 @@ class MovementEventHelper @Inject()(
+        }
+
+        buildOverviewPartial(
+          headingId = None,
+          headingTitle = Some(headingTitle),
+          cardTitleMessageKey = None,
+          cardFooterHtml = None,
+          summaryListRows = Seq(
+            Some(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorDetails.dateOfArrival", reportOfReceipt.dateOfArrival.formatDateForUIOutput())),
+            Some(summaryListRowBuilder(statusKey, messages(s"movementHistoryEvent.${event.eventType}.rorDetails.status.value.${reportOfReceipt.acceptMovement}"))),
+            Some(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorDetails.moreInformation", reportOfReceipt.otherInformation.getOrElse(s"movementHistoryEvent.${event.eventType}.rorDetails.moreInformation.notProvided"))),
+          )
+        )
+      case None => Html("")
+    }
+  }
+
+  def rorConsigneeCard(event: MovementHistoryEvent)(implicit movement: GetMovementResponse, messages: Messages): Html = {
+    val optContent: Option[Html] = for {
+      reportOfReceipt <- movement.reportOfReceipt
+      consigneeTrader <- reportOfReceipt.consigneeTrader
+    } yield {
+
+      val ernRow: Option[SummaryListRow] =
+        if (Seq(DestinationType.TaxWarehouse, DestinationType.DirectDelivery, DestinationType.RegisteredConsignee).contains(movement.destinationType)) {
+          consigneeTrader.traderExciseNumber.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorConsignee.ern", _))
+        } else {
+          None
+        }
+
+      val temporaryAuthorisationReferenceRow: Option[SummaryListRow] =
+        if (movement.destinationType == DestinationType.TemporaryRegisteredConsignee) {
+          consigneeTrader.traderExciseNumber.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorConsignee.temporaryReference", _))
+        } else {
+          None
+        }
+
+      buildOverviewPartial(
+        headingId = None,
+        headingTitle = Some(s"movementHistoryEvent.${event.eventType}.rorConsignee.h2"),
+        cardTitleMessageKey = None,
+        cardFooterHtml = None,
+        summaryListRows = Seq(
+          consigneeTrader.traderName.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorConsignee.name", _)),
+          ernRow,
+          temporaryAuthorisationReferenceRow,
+          consigneeTrader.vatNumber.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorConsignee.vatNumber", _)),
+          consigneeTrader.address.map(address => summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorConsignee.address", renderAddress(address))),
+          consigneeTrader.eoriNumber.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorConsignee.eoriNumber", _))
+        )
+      )
+    }
+
+    optContent.getOrElse(Html(""))
+  }
+
+  def rorDestinationCard(event: MovementHistoryEvent)(implicit movement: GetMovementResponse, messages: Messages): Html = {
```

Comment:
Couldn't the existing `Place of Destination` card be updated for this? I'd have thought this content would have been consistent everywhere?